app/components/common/eventCard/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Event } from '~/utils/interfaces';
+
+export default function EventCard({ name, attendees }: Event) {
+  return (
+    <>
+      <div className="border-2 w-64 h-72 bg-gray-200 rounded-3xl m-auto relative">
+        <div className="w-10 h-12 top-4 bg-white rounded-lg flex items-center flex-col absolute right-1 divide-y">
+          {/* Hard coding Date and month for time being, but will have to update once we have days.js in our codebase */}
+          <div className="text-black font-medium text-base">30</div>
+          <div className="text-black font-medium text-sm">AUG</div>
+        </div>
+
+        <div className="ml-6 bottom-4 gap-2 absolute flex flex-col">
+          {/* Hard coding Time for time being, but will have to update once we have days.js in our codebase */}
+          <div className="text-xs text-gray-700 font-medium">12AM to 1:30AM</div>
+          <div className="text-black font-medium text-lg event-title"> {name} </div>
```

Comment:
use `moment` right now. Don't hard code values. Later it can be converted to `dayjs`

Reply:
Updated, thanks!